specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,535 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "UserAssignedIdentities_Delete",
+        "description": "Deletes the identity.",
+        "x-ms-examples": {
+          "IdentityDelete": {
+            "$ref": "./examples/IdentityDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted Identity.",
+            "schema": {
+              "$ref": "#/definitions/Identity"
+            }
+          },
+          "204": {
+            "description": "The specified identity does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IdentityCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The supported Azure region where the associated Resource Group exists."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "Parameters for creating or updating identity."
+    },
+    "Identity": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the created identity."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the created identity."
+        },
+        "location": {
+          "type": "string",
+          "description": "The Azure region where the identity was created."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties associated with the identity.",
+          "$ref": "#definitions/IdentityProperties"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "UserAssignedIdentities"
```

Comment:
>UserAssignedIdentities [](start = 13, length = 22)

If the samples are correct, the value here should be "Microsoft.ManagedIdentity/userAssignedIdentities". I would guess that the x-ms-enum name value would be UserAssignedIdentities instead.

Reply:
You will also see that the model validator correctly has found this issue. I would strongly recommend that we make sure that all of the model validator tests are passing. If you specify the produces/consumes  see the [swagger documentation](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) for more information - it will fix a bunch of the current issues....

Reply:
Yes. I got the values from enum declaration and x-ms-enum(name) mixed up. Will change.

Reply:
Fixed.